openpower/package/hcode/Config.in

```diff
@@ -1,6 +1,7 @@
 config BR2_PACKAGE_HCODE
         bool "hcode"
         default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_POWER9)
+	depends on BR2_PACKAGE_HAS_PPE42_TOOLCHAIN
```

Comment:
tab indentation when rest of file is spaces

Reply:
yeah quite honestly I noted that but decided to ignore at the time considering our Makefiles and shell scripts are all over the place regarding tab vs space.. But I don't want to make it worse, so I'll fix these - thanks